tweet_recommendations/scripts/limit_preds.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+from pathlib import Path
+
+import pandas as pd
+import tqdm
+
+
+def limit_from_frame(frame: pd.DataFrame, limit: int) -> pd.DataFrame:
+    preds = frame.pred.tolist()
```

Comment:
odwołanie do kolumny przez kropkę - lepiej unikać :)

Reply:
A czemu w sumie? Nie spotkałem się z opinią, że lepiej unikać `dot notation`.

Reply:
Sam zawsze zwracałeś mi uwagę jak tego używałem :p
Generalnie dot notation nie zawsze może być wykorzystane, a [ ] może być zawsze. Do tego może nie być na pierwszy rzut oka oczywiste czy odnosi się to do kolumny czy do jakiegoś akcesora w klasie DataFrame. A skoro tak, to żeby mieć spójność w całym kodzie (w imię zen of python :p) lepiej stosować [ ].
Generalnie to zadziała i nie powinno robić problemów, tyle że nie kojarzę innych miejsc w kodzie, gdzie tak się odwoływaliśmy:)

Reply:
Nie kojarzę, żebym zwracał na to uwagę, ale ok :P